src/KMonad/App/KeyIO/Mac/IOKitSource.hs

```diff
@@ -86,3 +150,21 @@ withIOKitSource cfg = mkCtx $ \f -> do
                            (Keycode $ re^.reCode)
 
   bracket init cleanup $ \ptr -> f (nextEvent ptr)
+
+  where
+    print dp = do
+      dp <- peek dp
```

Comment:
Note to self: remove this.

Can this `where` section be moved to below `init` function? I am getting errors when I do that. I guess I am still not clear on where I can use `where`.